public/components/admin.js

```diff
@@ -194,7 +195,7 @@ class CardCategoryDocuments extends React.Component {
       let isRequired = (doc.required) ? 'required' : 'not required';
       docs.push(
         <li data-name={doc.name} key={i}>
-          <span className="card-category-document-name">{doc.name}</span>
+          <span id={doc.name} className="card-category-document-name">{doc.name}</span>
```

Comment:
The only reason I added it was for the nightwatch test that behaved strangely, but in the end I didn't use it, so it can be removed.